src/components/ZipCodeInput.js

```diff
@@ -1,42 +1,40 @@
-import { useState } from "react"
 import { Button, Stack, TextField } from "@mui/material"
 import SearchIcon from "@mui/icons-material/Search"
 import fetchPetList from "../components/fetchPetList"
 import { useContext } from "react"
 import { PetsContext } from "../context"
+import { SearchContext } from "../context"
 
 const ZipCodeInput = () => {
-  //using PetsContext
+  //using PetsContext and SearchContext
   const { setPetList } = useContext(PetsContext)
+  const { search, handleSearch } = useContext(SearchContext)
 
-  const [zipcodeFormValues, setZipcodeFormValues] = useState("")
-
-  //Gets value from the input and updates the zipCode state
+  //Validation. Gets value from the input and updates the zipCode state
   const handleZipcodeChange = (e) => {
     const userZipCode = e.target.value
     if (userZipCode.length <= 5) {
-      setZipcodeFormValues(userZipCode)
+      handleSearch(e)
     }
   }
-
-  //OnSubmit Form consoles the user input
+  //  makes a fetch request with current zip and selected animal type
+  //should handleSubmit be included in the SearchContext and moved to <App/>?
   const handleSubmit = (e) => {
```

Comment:
Should the `handleSubmit` function be moved to `<App/> `and added to `SearchContext`? Or should it just remain in `<ZipCodeInput/>`